frontend/src/metabase/components/MetadataInfo/TableInfo/TableInfo.tsx

```diff
@@ -59,7 +59,7 @@ function useDependentTableMetadata({
   fetchMetadata,
 }: Pick<AllProps, "tableId" | "table" | "fetchForeignKeys" | "fetchMetadata">) {
   const isMissingFields = !table?.numFields();
-  const isMissingFks = table?.fks == null;
+  const isMissingFks = table?.getPlainObject().fks == null;
```

Comment:
This should actually check `isLoaded` state for the corresponding entity query, but it would be too complicated to fix this right now